keras_maskrcnn/losses.py

```diff
@@ -3,75 +3,85 @@
 from . import backend
 
 
-def mask(iou_threshold=0.5, mask_size=(28, 28)):
-    def _mask(y_true, y_pred):
-        # split up the different predicted blobs
-        boxes = y_pred[:, :, :4]
-        masks = y_pred[:, :, 4:]
+def default_loss(y_true, y_pred, iou_threshold=0.5, mask_size=(28, 28)):
```

Comment:
Could we put this as a subfunction of `mask`, similar to `_mask` at the moment? Let's rename it to `_mask` and rename the current `_mask` to `_mask_conditional` (or something else if you have a better suggestion).